### ISSUE TITLE
InputTransparent with Opacity/IsVisible behaves differently with iOS/Mac vs Windows/Android

### ISSUE THREAD

**Issue by gktval** [dotnet/maui#18470]
### Description

I'd like to reopen this issue. https://github.com/dotnet/maui/issues/15608

I finally updated to .net 8 and got things to work on ios again. However, this issue was not fixed with the other IntputTransparent reworks https://github.com/dotnet/maui/pull/14846. I tested with the latest .Net8 rc.2.9511

@mattleibow 

### Steps to Reproduce

1.    Open the repository in the link in Mac/iOS. Click the "Toggle Bottom Sheet" button to open the Bottom Sheet. Then click the CounterBtn on the page and notice that you cannot click it.
 2.   Perform the above sequence and Windows/Android and observe that the CounterBtn can be clicked while the Bottom Sheet is open.

### Link to public reproduction project repository

https://github.com/gktval/MauiOpacityBug

### Version with bug

8.0.0-rc.2.9373

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

all versions

### Did you find any workaround?

No

**Comment by mattleibow** [comment]
Thanks for opening this issue. I had a look and I think maybe there is an incorrect property set: 

https://github.com/gktval/MauiOpacityBug/blob/f118c11271c4b73864cd5bec065579584977f9fc/BottomSheet.xaml.cs#L171

When the sheet is opened, this will set the background view to NOT be transparent. Was this intentional and performs some other action?

**Comment by gktval** [comment]
Thanks for taking a look at this so soon. The line you mention I think is the crux of the problem: When the sheet is opened, this should set the sheet itself to not be transparent anymore. Which means I should now be able to interact with the sheet and not interact with background on the underlying page.  However, background items in the transparent portion of the sheet should be clickable.  I tried placing the 'click me' button above the sheet when opened (so it is not directly underneath the opaque portion of the sheet). Since the button is in the transparent portion of the sheet, it should be clickable.

Here is the difference between the platforms:
1) In IOS, when the sheet is opened, I can no longer interact with the button in the transparent portion of the sheet (the upper half ish).
2) in Android/Windows, when the sheet is opened, I can still click the button that is above the sheet in the background page.

Preferably, I think that the Android/Windows is correct. This is how bottom sheets work in many other languages/platforms--that you can still interact with the elements that are not covered by the sheet.

**Comment by gktval** [comment]
Hey @mattleibow. I was able to work around this issue by changing the fill method of the bottom sheet to End instead of Fill. I tried reproducing the transparency issue in another project with grids/stacklayouts in the main page instead of a separate content view, but was unable to click on the background--so I am not really sure why it was different with the bottom sheet in this project.

Since I was able to find a workaround, I am not too worried if this is fixed. Seems like transparency stuff is working great after the previous fixes you made. All that to say, I will close this issue unless you are worried about it.
